From: Arnaud Ferraris <arnaud.ferraris@gmail.com>
Date: Sat, 20 Nov 2021 12:48:22 +0100
Subject: arm64: dts: pinephone: bump minimal backlight duty cycle

The previous value of 10% makes the display go fully black on some
devices. 11% seems to improve things, but it may still be too low for
some, so let's use 12% instead.
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts | 2 +-
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
index af9f223..121bdc0 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
@@ -21,7 +21,7 @@ &axp803 {
 
 &backlight {
 	power-supply = <&reg_ldo_io0>;
-	lth-brightness = <10>;
+	lth-brightness = <12>;
 };
 
 &codec_analog {
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
index 5030d78..034be04 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
@@ -27,7 +27,7 @@ &axp803 {
 
 &backlight {
 	power-supply = <&reg_ldo_io0>;
-	lth-brightness = <10>;
+	lth-brightness = <12>;
 };
 
 &codec_analog {
